regolith/schemas.py

```diff
@@ -1366,6 +1366,7 @@
             "schema": {
                 "type": "dict",
                 "schema": {
+                    "advisor": {"required": False, "type": "string"},
```

Comment:
I think a description could help here.

Also, we will need an examplar (above). Add it to an existing education scopatz item.